_gsocproposals/2020/proposal_IRIS-HEPhist.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Hist plotting
+layout: gsoc_proposal
+project: IRIS-HEP
+year: 2020
+organization: princeton
+---
+
+## Description
+
+The [Scikit-HEP](https://scikit-hep.org) project is a collection of several
+dozen packages intended to facilitate the use of Python in High Energy Physics
+(HEP), started in 2016. One of the major fronts of development is in
+histogramming; a majority of HEP analysis is heavily reliant on histograms. To
+this end, a new Python package was introduced for histogramming in Scikit-HEP,
+boost-histogram, developed in close collaboration with the author of the Boost
+C++ libraries' Histogram package. It is intended, however, to be a core package
```

Comment:
"It is intended, however, for it to be a core package" or "This package is intended" (maybe?)

Reply:
Thanks!